Assets/_Dev/Scripts/Manager/SoundManager.cs

```diff
@@ -4,15 +4,34 @@
 using UnityEngine;
 using UnityEngine.Audio;
 
-public class SoundManager : Singleton<SoundManager>
+public class SoundManager : MonoBehaviour, IAudioControl
 {
-    private Dictionary<eAudioMixerType, AudioSource> audioSourceDics = new Dictionary<eAudioMixerType, AudioSource>();
-    private Dictionary<eAudioClips, AudioClip> audioClipDics = new Dictionary<eAudioClips, AudioClip>();
+    [field: SerializeField] public AudioSource audioSource { get; set; }
+    [field: SerializeField] public AudioClip audioClip { get; set; }
```

Comment:
property는 대문자로 시작합니다.
```suggestion
    [field: SerializeField] public AudioSource AudioSource { get; set; }
    [field: SerializeField] public AudioClip AudioClip { get; set; }
```

Reply:
프로퍼티 대문자, 다시한번 메모리!